test/unit/org/apache/cassandra/cql3/conditions/ColumnConditionTest.java

```diff
@@ -321,27 +321,28 @@ public void testListCollectionBoundAppliesTo() throws InvalidRequestException
 
         // LT
         assertFalse(conditionApplies(list(ONE), LT, list(ONE)));
+        assertThrowsIRE(() -> conditionApplies(list(), LT, null), "Invalid comparison with null for operator \"<\"");
```

Comment:
This is fix, because before we were throwing `NullPointerException`?

Reply:
For multicell collection an empty collection is the same as null. I wanted to ensure that the behaviour was correct.